recipes/log4cxx/all/conandata.yml

```diff
@@ -1,14 +1,75 @@
 sources:
+  "1.1.0":
+    sha256: "1fc7d82697534184bc0f757348d969d24852b948f63d6b17283fd1ee29c2c28a"
+    url: "https://archive.apache.org/dist/logging/log4cxx/1.1.0/apache-log4cxx-1.1.0.tar.gz"
+  "1.0.0":
+    sha256: "6df9f1f682650de6045309473d5b2fe1f798a03ceb36a74a5b21f5520962d32f"
+    url: "https://archive.apache.org/dist/logging/log4cxx/1.0.0/apache-log4cxx-1.0.0.tar.gz"
+  "0.13.0":
+    sha256: "4e5be64b6b1e6de8525f8b87635270b81f772a98902d20d7ac646fdf1ac08284"
+    url: "https://archive.apache.org/dist/logging/log4cxx/0.13.0/apache-log4cxx-0.13.0.tar.gz"
```

Comment:
Do you really need these 2 versions? Since there is 1.1.0, and there are many patches in 1.0.0 & 0.13.0, I'm not sure it's worth the effort and maintenance burden.

Reply:
Version 1.1.0 is sufficient - 0.13.0 has a different ABI and fixes some 0.12.0 mutli-threading performance issues.